docs/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway.md

```diff
@@ -24,10 +24,12 @@ Follow the instruction to expose and access your unsecured instance of the HttpB
 1. Export the following value as an environment variable:
 
    ```bash
-   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} #This is a Kyma domain or your custom subdomain e.g. api.mydomain.com.
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} #This is a Kyma domain or your custom subdomain, for example, api.mydomain.com
+   export GATEWAY=$NAMESPACE/httpbin-gateway
    ```
+   >**NOTE:** If you don't want to use your custom domain but a Kyma domain, use the following Kyma Gateway: `kyma-system/kyma-gateway`
```

Comment:
```suggestion
   >**NOTE:** If you want to use Kyma domain, export it as `DOMAIN_TO_EXPOSE_WORKLOADS` and use the following Kyma Gateway: `kyma-system/kyma-gateway`
```
or something similar. this way we will get rid of suggestions in comments. Please change that consistently everywhere